src/router.js

```diff
@@ -1,12 +1,13 @@
 import Vue from "vue";
 import Router from "vue-router";
 import Home from "pages/Home.vue";
```

Comment:
Can we remove Home and maybe the Landing folder?

Reply:
Ya I will not sure why the home is in there. Do you not want folders inside the pages folder just files?

Reply:
Just files, .vue files can have the style, js, and template all in one file so should probably just do that, the styles folder is just for generic selectors we will use for our personal "bootstrap" css